ballerina/uuid.bal

```diff
@@ -342,3 +344,43 @@ public isolated function toRecord(string|byte[] uuid) returns Uuid|Error {
     };
     return uuidRecord;
 }
+
+# Represents a UUID instance.
+class UUID {
```

Comment:
is it correct to name this as `UUID` as it only contains the least and most significant bits?

Reply:
A `UUID` which is a 128-bit label is comprised with `leastSigBits` and `mostSigBits` which are 64 bit each. The other fields like `timeLow`, `timeMid`, etc. are derived from `leastSigBits` and `mostSigBits`. Therefore, I think it's okay to name it `UUID`.

These 2 are the only fields in Java's UUID class[[1]](https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html) as well.

Another option is to name it something like `UuidMeta`. WDYT?

[1] https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html

Reply:
As discussed, removed the class in [d5f025a](https://github.com/ballerina-platform/module-ballerina-uuid/pull/374/commits/d5f025a7e5d2644bceb1b97861a46a273cc6ca51)